ledger-core/v2/Makefile.testing

```diff
@@ -14,13 +14,12 @@ test_unit: ## run all unit tests
 
 .PHONY: test_func
 test_func: ## run functest FUNCTEST_COUNT times
-	CGO_ENABLED=1 $(GOTEST) -test.v $(TEST_ARGS) -tags "functest bloattest" ./application/functest -count=$(FUNCTEST_COUNT)
+	CGO_ENABLED=1 $(GOTEST) -test.v $(TEST_ARGS) -tags "functest" ./application/functest -count=$(FUNCTEST_COUNT)
 
 .PNONY: test_func_race
 test_func_race: ## run functest 10 times with -race flag
 	make clean
-	GOBUILD='go build -race' make build
```

Comment:
functests use launchnet and ignore this pre-build.

Reply:
previously launchnet doesn't ignore it

Reply:
It builds now with the right flags?

Reply:
Added `race` flag for build and for tests